test/support/generator/get-geospatial-addresses-generator.ts

```diff
@@ -0,0 +1,230 @@
+import { ENUMS, GEOSPATIAL } from '@ukef/constants';
+import { GetAddressesOrdnanceSurveyResponse } from '@ukef/helper-modules/ordnance-survey/dto/get-addresses-ordnance-survey-response.dto';
+import { OrdnanceSurveyAuthErrorResponse } from '@ukef/helper-modules/ordnance-survey/dto/ordnance-survey-auth-error-response.dto';
+import { GetAddressesByPostcodeQueryDto } from '@ukef/modules/geospatial/dto/get-addresses-by-postcode-query.dto';
+import { GetAddressesResponse } from '@ukef/modules/geospatial/dto/get-addresses-response.dto';
+
+import { AbstractGenerator } from './abstract-generator';
+import { RandomValueGenerator } from './random-value-generator';
+
+export class GetGeospatialAddressesGenerator extends AbstractGenerator<AddressValues, GenerateResult, GenerateOptions> {
+  constructor(protected readonly valueGenerator: RandomValueGenerator) {
+    super(valueGenerator);
+  }
+
+  protected generateValues(): AddressValues {
+    return {
+      ORGANISATION_NAME: this.valueGenerator.sentence({ words: 2 }),
+      BUILDING_NAME: this.valueGenerator.word(),
+      BUILDING_NUMBER: this.valueGenerator.nonnegativeInteger().toString(),
+      THOROUGHFARE_NAME: this.valueGenerator.sentence({ words: 5 }),
```

Comment:
```suggestion
      ORGANISATION_NAME: this.valueGenerator.sentence({ words: 5 }),
      BUILDING_NAME: this.valueGenerator.word(),
      BUILDING_NUMBER: this.valueGenerator.nonnegativeInteger().toString(),
      THOROUGHFARE_NAME: this.valueGenerator.sentence({ words: 2 }),
```

Reply:
Very minor thing but I think the number of words for `ORGANISATION_NAME` and `THOROUGHFARE_NAME` might be the wrong way round - it would make more sense for a throughfare (i.e., street) to be 2 words and an organisation to be 5 words I think

Reply:
I expected THOROUGHFARE_NAME to be longer, but yes, it is just street and usually is short.